advanced/Templates/gravity.db.sql

```diff
@@ -1,3 +1,12 @@
+PRAGMA FOREIGN_KEYS=ON;
+
+CREATE TABLE "group"
+(
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	enabled BOOLEAN NOT NULL DEFAULT 1,
+	description TEXT
```

Comment:
Does it make sense to have both a required `name` and optional `description` field? Right now, it is possible to create a group without any useful identifier other than the ID.